src/config/webpack.config.js

```diff
@@ -147,6 +147,9 @@ const defaultWebpackConfig = {
   resolveLoader: {
     modules: [path.resolve(path.join(rootPath, 'node_modules'))],
   },
+  node: {
+    fs: 'empty',
```

Comment:
What exactly is this doing and why? It seems pretty shady.

Reply:
https://github.com/cerner/terra-framework/blob/embedded-component/packages/terra-embedded-component/docs/README.md#troubleshooting

Reply:
Also https://webpack.js.org/configuration/node/#other-node-core-libraries

Reply:
Following the issue linked in the embedded-component readme, it was posted a better alternative to included 
```
node: {
    fs: 'empty',
}
```
would be:
```
const nodeExternals = require('webpack-node-externals');

{
  target: 'node',
  externals: [nodeExternals()],
}
```
[Mentioned here](https://stackoverflow.com/questions/41692643/webpack-and-express-critical-dependencies-warning/42425214#42425214).

I would be interested in exploring https://www.npmjs.com/package/webpack-node-externals to see if this is a better/feasible solution to the issue. However, IMO changes to the webpack in this sense will need to be scoped and tested with each repo & the theming-repo setup to ensure each site is build correctly. My concern would be with the theming repos.

Reply:
I'm gathering that this is setting the fs reference to empty which seems like it might have other side effects on our build. I'm wondering if we could try some of the other solutions mentioned here. https://github.com/webpack-contrib/css-loader/issues/447.
Also this issue is marked as resolved. Do we need to upgrade our css-loader version?

Reply:
+1 to @mjhenkes I found the link I posted from the https://github.com/webpack-contrib/css-loader/issues/447 discussion. The issue was closed with mentioned this was actually an issue with webpack itself, not necessarily the loader, so I do not think updating will be the solution. 

![screen shot 2018-02-23 at 11 59 37 am](https://user-images.githubusercontent.com/14099737/36609079-60449dac-1891-11e8-80a5-df68c7ef331c.png)

Reply:
Though, digging deeper, it seems it may be an issue with css-loader itself. https://github.com/webpack/webpack/issues/4458

Reply:
I tried using the webpack-node-externals module and its throwing the following browser error: 
![screen shot 2018-02-23 at 2 38 26 pm](https://user-images.githubusercontent.com/12060103/36616119-3732cc2e-18a8-11e8-9115-efe182edc115.png)

I'm not sure if there is a more specific way of using the module. I just set it up without anything whitelisted (as the 'quick-usage' section does here: https://www.npmjs.com/package/webpack-node-externals)

Reply:
The issue is the SystemJS library when compiled with webpack has code which is looking the 'fs' module which makes no sense in the web world (its a single code base for web and node). This change shouldn't impact any of our other code, because none of our other code is using the fs module.

Reply:
Think I found the root cause. Webpack by default polyfills all the node APIs (or mocks them if it can't). We disabled this behavior by default when we set https://github.com/cerner/terra-dev-site/blob/master/src/config/webpack.config.js#L148 and removed the defauls provided by webpack such as `web_modules` which profiles this behavior. https://github.com/webpack/docs/wiki/Configuration#resolveloader

@grneggandsam try adding `web_modules` into the resolveLoader configuration and see if that fixes it.

Reply:
Unfortunately I wasn't able to get the error resolved using the resolveLoader or any of its defaults. The only thing that seems to be working so far is the node: {fs: 'empty} solution

Reply:
With the latest commit to Terra-Framework (https://github.com/cerner/terra-framework/pull/91/commits/20d325d408861acae6bdabb65f75c5430a71a047) the issue with the 'fs' module was resolved with the embedde-component's import, so this change is no longer needed.